src/main/java/seedu/duke/Ui.java

```diff
@@ -364,7 +364,7 @@ public void showCheckClientNoPair(Client client) {
     //@@author
 
     //@@author ngdeqi
-    public void showCheckProperty(Property property, ArrayList<Client> tenants) {
+    public void showCheckPropertyMessage(Property property, ArrayList<Client> tenants) {
```

Comment:
The new function is indeed more appropriate and helps reader understand the function better.